client/modules/workspace/src/AppsWizard/AppsFormSchema.ts

```diff
@@ -186,6 +189,27 @@ export const getConfigurationFields = (
     definition.jobAttributes.execSystemId
   ) as TTapisSystem;
 
+  if (definition.jobType === 'BATCH') {
+    const systemOptions = execSystems.map((sys) => ({
+      value: sys.id,
+      label: sys.id,
```

Comment:
What's the difference between value and label here? Is there a more specific name other than value?